openedx/openedx-platform
# feat: add taxonomies for org api

## Description
This PR adds support for creating, viewing, updating and deleting Taxonomies for Organizations via REST APIs, with the permissions applied accordingly.

## Supporting information

* Depends on https://github.com/openedx/openedx-learning/pull/63
* Closes https://github.com/openedx/modular-learning/issues/74

## Testing instructions

* Check the unit tests in `test_views.py` and `test_rules.py` are correct.
* Run the unit tests

Manual check can be done in the CMS App:
- Run the cms app
- Login using Django Admin (non-staff users can login, they just can't see Django Admin)
  - http://127.0.0.1:8000/admin/
- REST API is here:
  - http://127.0.0.1:8000/api/content_tagging/v1/

---
Private-ref: [FAL-3450](https://tasks.opencraft.com/browse/FAL-3450)
